Dockerfile

```diff
@@ -15,7 +15,7 @@
 # We copy files from the context into a scratch container first to avoid a problem where docker and
 # docker-compose don't share layer hashes https://github.com/docker/compose/issues/883 normally.
 # COPY --from= works around the issue.
-FROM scratch as scratch
```

Comment:
the precise problem here only happens in this image, as later we do this:

```
FROM scratch as alpine
```

there's ambiguity on what that wants to be, and it grabs the layer named scratch not the actual scratch

Reply:
what this caused was lingering of the script `/code/alpine_minirootfs`, which then caused me to be severely confused troubleshooting later as wondering how in the heck that got there (from a dependent image)

lesson learned.. never call things scratch, there should be only one scratch